.ci/deps.sh

```diff
@@ -50,7 +53,6 @@ if [ "$USE_PPAS" = "true" ]; then
 fi
 
 deps_perl="perl libperl-critic-perl"
-deps_infer="m4 opam"
 
 sudo apt-get -y update
 sudo apt-get -y --no-install-recommends install $deps $deps_perl $deps_infer
```

Comment:
No. `$deps_infer` was moved earlier in the script, and is modified on xenial. `deps_infer="$deps_infer ocaml camlp4"`